crates/agglayer-storage/src/columns/proof_per_certificate/tests.rs

```diff
@@ -40,8 +40,6 @@ fn can_parse_value() {
 #[derive(Deserialize)]
 struct SP1ProofWithPublicValuesV3 {
     pub proof: sp1_sdk::SP1Proof,
-    #[allow(unused)]
-    pub stdin: sp1_sdk::SP1Stdin,
```

Comment:
I don't think this change is correct. It changes the serialization format which is used to load the values of this type from files.

Reply:
Thank you for the feedback.

This specific field now has a "reason" declaration to avoid future accidental removals